src/main/webapp/WEB-INF/tags/ksr/securityRequestQualifierLookup.tag

```diff
@@ -0,0 +1,31 @@
+  <%@ include file="/jsp/sys/kfsTldHeader.jsp" %>
+
+<%@ attribute name="requestQualifications" required="true" type="java.util.List"%>
+<%@ attribute name="pathPrefix" required="true"%>
+<%@ attribute name="attributeDefinition" required="true" type="org.kuali.kfs.kim.api.type.QuickFinder"%>

```

Comment:
Since this attribute is a QuickFinder and not an AttributeDefinition, should it be renamed to avoid confusion?

Reply:
updated